src/router/index.js

```diff
@@ -4,6 +4,7 @@ import Start from '../components/Start';
 import Account from '../components/Account';
 import Block from '../components/Block';
 import Transaction from '../components/Transaction';
+import Transactions from '@/components/Transactions';
```

Comment:
Since we have this alias now, we should replace the  `..` at least in the files, we're editing